src/base/LMIcon/types.tsx

```diff
@@ -0,0 +1,13 @@
+import { ImageStyle, ViewStyle } from "react-native";
+
+export interface LMIconProps {
+    type: 'png',
+    assetPath?: string,
+    color?: string,
+    size?: number,
```

Comment:
height and width can be different for a image

Reply:
resolved